code/__DEFINES/dcs/signals/signals_mob/signals_mob_carbon.dm

```diff
@@ -132,3 +132,5 @@
 
 ///From mob/living/carbon/human/suicide()
 #define COMSIG_HUMAN_SUICIDE_ACT "human_suicide_act"
+///From mob/living/carbon/human/suicide(), called when a NON-MANUAL suicide is triggered.
```

Comment:
What does "non manual" mean here?

Reply:
if the suicide isnt of the type MANUAL_SUICIDE or MANUAL_SUICIDE_NONLETHAL

Reply:
I see, include that in the documentation